packages/kbn-axe-config/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/axe-config
+
+Empty package generated by @kbn/generate
```

Comment:
Suggested README documentation:

```suggestion
This is a package shares [axe](https://www.deque.com/axe/) rule configuration and options between various different axe tests and test runners (e.g., Kibana FTR, Cypress, jest). The API remains the same between each axe runner, and should ideally be shared between each to maintain consistency across Kibana.
```

Reply:
Thank you!

Reply:
Also wanted to say thanks for taking care of this and finding the issue Spencer! When I split out the axe config from Kibana's FTR way back my goal was simply to DRY out shared/standard axe rules, and I totally didn't realize it would end up causing build shenanigans 🙇